src/ctap2/attestation.rs

```diff
@@ -338,19 +342,31 @@ impl From<&[u8]> for Signature {
     }
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq, Deserialize)]
+#[serde(tag = "fmt", content = "attStmt", rename_all = "lowercase")]
```

Comment:
Could you maybe add an explanatory comment here, what exactly this does? So one doesn't have to necessarily read the serde-docs to get a rough idea of whats happening.

Reply:
Ah, it's explained below. Nevermind then. Maybe point to that explanation, but I think it's fine to leave it as is.

Reply:
It is a bit confusing. I added a comment to AttestationStatement as well.